### ISSUE TITLE
Add react widget to FMP page for incremental release

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#88012]

**Comment by laurwill** [comment]
Hi @marywang2 , questions here as we start the process for incremental release:
- You mentioned that you already have the react widget ID — is there already content coded in the widget and if so can you post that content here?
- What's the target date to start incremental release of this form?

**Comment by marywang2** [comment]
Hi @laurwill, here is the ID: form107959F1. I don't believe there is any content there - do you have any guidance for us (as first timers)? Target date is TBD but sometime in August possibly?

**Comment by laurwill** [comment]
Hi @marywang2 , that's fine, I'll provide the content for both sides of the widget! I can do that this week, and then we can connect about testing the widget in the page.

**Comment by marywang2** [comment]
Awesome, thanks so much @laurwil! Yes will definitely keep you posted.

**Comment by laurwill** [comment]
Hi @marywang2 , do you know if the FMP office wants the online form to replace the option to upload the pdf form through Ask VA? Or do we need to continue to offer that option?
<img width="503" alt="image" src="https://github.com/user-attachments/assets/28f9481b-a865-493c-8e6e-0c60acebccbb">

**Comment by marywang2** [comment]
Hmm that's a good question. We should probably check with stakeholders. We have a meeting next week so I can ask them then. cc @brian-wilke for awareness.

**Comment by pshahVA** [comment]
From what I've seen the upload option is only on forms that don't have an online experience. I'd like to turn off that upload option  as part of taking this live.  @laurwill do you have any examples where this isn't the case?

**Comment by laurwill** [comment]
Nope, I don't know of any, and removing the upload option when you launch the online form makes sense to me.

**Comment by marywang2** [comment]
@pshahVA @laurwill the above sounds good to me - we can follow the standards set w/other online forms.

**Comment by laurwill** [comment]
Hi @marywang2 , here's content for the 2 sides of the react widget. We'll place the widget below the H2 "How to register" on [this page](https://www.va.gov/health-care/foreign-medical-program/).

### Side 1: No link to online form

Fill out an FMP Registration Form (VA Form 10-7959f-1). You’ll need your Social Security number or your VA claim number (also called VA file number). In most cases, your claim number and Social Security number are the same.

**Note:** You don’t need to be enrolled in VA health care to register for the Foreign Medical Program.

[primary action link] [Get VA Form 10-7959f-1 to download](https://www.va.gov/find-forms/about-form-10-7959f-1)

Then send your completed form to us in one of these ways.

**[H3] Option 1: Online**
[Upload your completed form through Ask VA](https://ask.va.gov/)

**[H3] Option 2: By mail**
Mail your completed form to this address:
> VHA Office of Integrated Veteran Care (OIVC) 
> Foreign Medical Program (FMP)
> P.O. Box 469061
> Denver, CO 80246-9061

**[H3] Option 3: By fax**
Fax your completed form to [303-331-7803](tel:+1303-331-7803).

### Side 2: Link to online form

You can register online, by mail, or by fax. You’ll need your Social Security number or your VA claim number (also called VA file number). In most cases, your claim number and Social Security number are the same.

**Note:** You don’t need to be enrolled in VA health care to register for the Foreign Medical Program.

[H3] **Option 1: Online**
[Primary action link] [Register for the Foreign Medical Program](URL for intro page of online form)

**[H3] Option 2: By mail**
Fill out an FMP Registration Form (VA Form 10-7959f-1).

[Get VA Form 10-7959f-1 to download](https://www.va.gov/find-forms/about-form-10-7959f-1)

Mail your completed form to this address:
> VHA Office of Integrated Veteran Care (OIVC)
> Foreign Medical Program (FMP)
> P.O. Box 469061
> Denver, CO 80246-9061

**[H3] Option 3: By fax**
Fill out an FMP Registration Form (VA Form 10-7959f-1).

[Get VA Form 10-7959f-1 to download](https://prod.cms.va.gov/find-forms/about-form-10-7959f-1)

Fax your completed form to [303-331-7803](tel:3033317803).

**Comment by laurwill** [comment]
Drupal preview isn't working right now, but I'll get you screenshots to share with your developers as soon as it is!

**Comment by laurwill** [comment]
@marywang2 

### Screenshot for Side 1:

![Image](https://github.com/user-attachments/assets/d589b40a-b804-463b-acbc-38200c7c27df)

### Screenshot for Side 2:

![Image](https://github.com/user-attachments/assets/b2707eac-b631-4e24-9a7d-f76cd90e5d57)

**Comment by laurwill** [comment]
Widget is published in the page, moving to validate
